src/source/documentUploader.ts

```diff
@@ -10,7 +11,7 @@ export async function uploadBatch(
   strategy: UploadStrategy,
   batch: BatchUpdateDocuments,
   createFields = true
-) {
+): Promise<AxiosResponse> {
   if (createFields) {
     const analyser = new FieldAnalyser(platformClient);
```

Comment:
I just noticed, tho reallly minor.
Which 'flavoor' of English is preferred for public APIs?
British (so -ise) or American (-ize)?

Reply:
British or course 
![image](https://user-images.githubusercontent.com/12199712/160848756-7fb62304-7400-4641-8e6f-67d3b762d68a.png)